### ISSUE TITLE
Blocked By Autoguard/Parrying/Kyrie Eleison.. all melee and normal attack should only miss but should not receive any reflect damage

### ISSUE THREAD

**Issue by BringerxD** [rathena/rathena#3480]
* **rAthena Hash**: ce9cbd2

* **Client Date**: 2013-12-23

* **Server Mode**: Pre-renewal

* **Description of Issue**: Sample: If Lord Knight is buffed by Parrying and it activated when hit by Champion's Asura Strike, therefore Champ Asura strike is blocked by Parrying and will miss but Champion character should not receive any reflect damage at all. In my case, it will miss and will also receive the reflect damage, which should not be. 

  * How to Reproduce:  Use +10 Valkyrian Manteau HIT player under KE / auto guard / parrying

**Comment by tanlor** [comment]
Still reflecting on last rAthena version. It reflects on Kyrie Eleison too.

**Comment by Immortals-RO** [comment]
That's the official 'Behaviour'.

**Comment by BringerxD** [comment]
> That's the official 'Behaviour' better play on Official so you should know how it works. or else you have to modify your own **SRC side**.

i want to revert the setting but i cant find it the pull

**Comment by alexandredrf** [comment]
anybody knows to revert this ?

**Comment by aleos89** [comment]
Duplicate of #2671 and fixed in #3129.
